lib/searchable_select.html.heex

```diff
@@ -95,6 +83,31 @@
               </div>
             </div>
           <% end %>
+          <%= if @limit_hit? && @limit_hit_text do %>
+            <div class="overflow-none shrink-0">
```

Comment:
```suggestion
```
I think having all these `divs` is a bit superfluous. I tried my end and simplified this whole block to:
`              <div class="overflow-none shrink-0" id={"#{@id}-remove-limit-option"} phx-click="remove_limit" phx-target={@myself}>
                <div class="cursor-pointer w-full border-gray-200 border-b hover:bg-primary-400 hover:text-white">
                    <div class="flex w-full items-center p-2 pl-2 border-transparent border-l-2 relative justify-between">
                      <strong><%= @limit_hit_text %></strong>
                    </div>
                </div>
              </div>`
With it all looking good still.